src/components/Header/Tools.tsx

```diff
@@ -23,7 +23,7 @@ import { createSupportCase } from '../../utils/createCase';
 import BellIcon from '@patternfly/react-icons/dist/dynamic/icons/bell-icon';
 import ChromeAuthContext from '../../auth/ChromeAuthContext';
 import { isPreviewAtom, togglePreviewWithCheckAtom } from '../../state/atoms/releaseAtom';
-import { notificationDrawerExpandedAtom, unreadNotificationsAtom } from '../../state/atoms/notificationDrawerAtom';
+import { notificationDrawerExpandedAtom, notificationDrawerUnreadAtom } from '../../state/atoms/notificationDrawerAtom';
```

Comment:
I do not think this atom should exist in the chrome repository. I'd move the Notification button to the Notifications repository as well.

With the singleton module, the drawer does not need any state in Chrome with the exception of initializing it.